### PR TITLE
Update service registry

### PR DESCRIPTION
**Pull request by Adamantios**
## Proposed changes

- Updates the service registry json.
- Updates the `contract.py` to work with the updated service registry. Method `getServiceInfo` does not exist anymore. We are now using `getAgentInstances` instead.
- Updates registration behaviour in order to use the new contract method.
- Fixes unit tests.
- Refactors the contract's unit tests to **not** use `Ganache` and deploy the contract.

## Fixes

n/a

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [x] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

1c63f108b554eb272bbb943d46be75630186178d requires attention. I believe that @Karrenbelt and @8ball030 have an open ticket for that.

### CODE REVIEW

File: packages/valory/agents/registration_start_up/aea-config.yaml [hunk 1]
```diff
@@ -93,18 +93,9 @@ public_id: valory/p2p_libp2p_client:0.1.0
...
   - uri: ${P2P_URI:str:acn.staging.autonolas.tech:9006}
     public_key: ${P2P_PUBLIC_KEY:str:02e741c62d706e1dcf6986bf37fa74b98681bc32669623ac9ee6ff72488d4f59e8}
 cert_requests:
-- identifier: acn
-  ledger_id: ethereum
-  message_format: '{public_key}'
-  not_after: '2023-01-01'
-  not_before: '2022-01-01'
-  public_key: ${P2P_PUBLIC_KEY:str:02d3a830c9d6ea1ae91936951430dee11f4662f33118b02190693be835359a9d77}
-  save_path: .certs/acn_cosmos_9005.txt
```
Comment (Adamantios) [inline packages/valory/agents/registration_start_up/aea-config.yaml:107]: This requires attention. Please take a look at the "Further Comments" section of the PR's description.

**Review by 8ball030 — APPROVED** [review]
LGTM

**Comment by dvilelaf** [comment]
What's up with [tests/test_protocols/test_abci/test_abci_fuzz.py::TestAeaHypotheses::test_request_init_chain](url) in macos?

**Review by dvilelaf — COMMENTED** [review]
Looks good. It seems we have a failing fuzzy test.

File: packages/valory/agents/registration_start_up/aea-config.yaml [hunk 2]
```diff
@@ -93,18 +93,9 @@ public_id: valory/p2p_libp2p_client:0.1.0
 type: connection
 config:
   nodes:
```
Comment (DavidMinarsch) [inline packages/valory/agents/registration_start_up/aea-config.yaml:95]: Why removed?

File: packages/valory/contracts/service_registry/contract.py [hunk 3]
```diff
@@ -49,6 +50,27 @@ class ServiceRegistryContract(Contract):
 
     contract_id = PUBLIC_ID
```
Comment (DavidMinarsch) [inline packages/valory/contracts/service_registry/contract.py:52]: Why all the not implemented methods?

Comment (Adamantios) [reply packages/valory/contracts/service_registry/contract.py:52]: We implement the abstract methods but raise since we do not need them.

**Review by DavidMinarsch — CHANGES_REQUESTED** [review]
Please address my comments for my understanding

**Comment by Adamantios** [comment]
> What's up with [tests/test_protocols/test_abci/test_abci_fuzz.py::TestAeaHypotheses::test_request_init_chain](url) in macos?

This is unrelated to the changes on this PR. 
```
hypothesis.errors.FailedHealthCheck: Data generation is extremely slow: Only produced 6 valid examples in 1.00 seconds (0 invalid ones and 0 exceeded maximum size). Try decreasing size of the data you're generating (with e.g. max_size or max_leaves parameters)
```

**Pull request merged**